packages/dashboard/src/features/analytics/components/AnalyticsLayout.tsx

```
@@ -0,0 +1,98 @@
+import { useEffect } from 'react';
+import { Outlet } from 'react-router-dom';
+import { useQueryClient } from '@tanstack/react-query';
+import { ErrorState, LoadingState } from '#components';
+import { useDashboardHost } from '#lib/config/DashboardHostContext';
+import { useProjectId } from '#lib/hooks/useMetadata';
+import { useToast } from '#lib/hooks/useToast';
+import { TimeRangeProvider } from '../context/TimeRangeContext';
```

ESLint

[error] '../context/TimeRangeContext' import is restricted from being used by a pattern. Use relative imports only within the same folder. Use dashboard #... aliases for imports outside the current folder.

(no-restricted-imports)